src/sentry/runner/commands/presenters/presenterdelegator.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any
+
+from sentry.runner.commands.presenters.consolepresenter import ConsolePresenter
+from sentry.runner.commands.presenters.optionspresenter import OptionsPresenter
+from sentry.runner.commands.presenters.slackpresenter import SlackPresenter
+
+
+class PresenterDelegator(OptionsPresenter):
+    def __init__(self, dry_run) -> None:
+        self.consolepresenter = ConsolePresenter(dry_run)
+        if self.check_slack_webhook_config():
+            self.slackpresenter = SlackPresenter(dry_run)
+        self.slackpresenter = None
+
+    def check_slack_webhook_config(self):
+        return True
```

Comment:
This does not seem in the right place. This way the delegator would have to know how to initialize slack.
Please move this to a static method in the `SlackPresenter`

Reply:
I think this is still in the wrong place.
Also what I meant was to turn it into a static method so you can do this in the `__init__` method

```
def __init__(self) -> None:
        self.consolepresenter = ConsolePresenter()
        if SlackPresenter.is_slack_enabled():
             self.slackpresenter = SlackPresenter()
        else:
            self.slackpresenter = None
```